client/src/components/RelatedItemsAndComparisons/Card.jsx

```diff
@@ -22,15 +30,64 @@ function Card({
     // comparing current product to selected product
   };
 
+  const findDefault = (prodStyles) => {
+    const defaultImgs = [];
+    for (let i = 0; i < prodStyles.length; i += 1) {
+      if (prodStyles[i]['default?']) {
+        setSalePrice(prodStyles[i].sale_price);
+        prodStyles[i].photos.forEach((img) => {
+          defaultImgs.push(img.thumbnail_url);
+        });
+      }
+    }
+    setDefaultImages(defaultImgs);
+  };
+
+  // =============GET REQUEST FOR STYLES================
+  const styles = axios.get(`/products/${product.id}/styles`);
+
+  // TODO: send another request to get product ratings
+  useEffect(() => {
+    axios.all([styles])
+      .then(axios.spread((style) => {
+        console.log('prod styles: ', style.data);
+        findDefault(style.data);
+        setProductStyles(style.data);
+      }))
+      .catch((err) => {
+        console.error('Couldnt get styles...', err);
+      });
+  }, []);
+
   // TODO: replace button with star icon
   return (
     <div className="productCard" onClick={handleCardClick}>
       <button className="compareButton" type="submit" onClick={handleCompareClick}>Star</button>
-      <img className="cardImg" src={tempImg} alt="product" />
+      <img className="cardImg" src={defaultImages[0] || defaultImgURL} alt="product" />
```

Comment:
This is the reason why I suggest renaming defaultImages, look weird since you have an actual defaultImgURL, to be naming the legit url a defaultImage